standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/DatabaseProduct.java

```diff
@@ -675,6 +676,24 @@ public List<String> createInsertValuesStmt(String tblColumns, List<String> rows,
     }
   }
 
+  public String createUpdatePreparedStmt(String tableName, List<String> columnNames,
```

Comment:
why it's in DatabaseProduct, they are generic and not db specific? should we move them to TxnUtils and mark them as static?

Reply:
Make sense.